components/network/weechat/Makefile

```diff
@@ -0,0 +1,74 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"). You may
+# only use this file in accordance with the terms of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright (c) 2020 Evgeny Litvinenko
+#
+
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=		weechat
+COMPONENT_VERSION=	2.8
+COMPONENT_PROJECT_URL=	https://weechat.org/
+COMPONENT_SUMMARY=	WeeChat - portable, fast, light and extensible IRC client
+COMPONENT_DESCRIPTION=	WeeChat (Wee Enhanced Environment for Chat) is a portable, fast, light and \
+extensible chat client. Everything can be done with a keyboard. \
+It is customizable and extensible with scripts.
+COMPONENT_SRC=		$(COMPONENT_NAME)-$(COMPONENT_VERSION)
+COMPONENT_ARCHIVE=	$(COMPONENT_SRC).tar.xz
+COMPONENT_ARCHIVE_HASH=	\
+    sha512:3bebb42118fca6b601f1de8f4d52378619a10e5c490f5b46f86d27a9680f00a56d48a1ade729e78d965cae589e6c80647c22414393000c7d3cfcc94a1e41c4df
+COMPONENT_ARCHIVE_URL=	https://weechat.org/files/src/$(COMPONENT_ARCHIVE)
+COMPONENT_SIG_URL=	$(COMPONENT_ARCHIVE_URL).asc
+COMPONENT_FMRI=		network/chat/weechat
+COMPONENT_LICENSE=	GPLv3
+COMPONENT_LICENSE_FILE=	$(COMPONENT_NAME).license
+COMPONENT_CLASSIFICATION=	Applications/Internet
+
+include $(WS_MAKE_RULES)/prep.mk
```

Comment:
Can you switch to common.mk (like in https://github.com/OpenIndiana/oi-userland/blob/oi/hipster/components/desktop/ekiga/Makefile)?
Include only common.mk (instead of prep,configure,ips.mk), and you can remove lines 53-59

Reply:
done